scripts/build_versioned_research_data_package.py

```
@@ -221,6 +334,207 @@ def _load_variable_and_value_labels(
+    ("manually_reviewed", "Manually reviewed and accepted"),
+    ("rejected", "Rejected after review"),
+]
+
+_SCHEMA_V2_VALUE_LABELS: list[dict[str, str]] = (
+    # evidence_fragments.source_field
+    [_v2vl(_EF, "source_field", "title", "Title field")]
+    + [_v2vl(_EF, "source_field", "subject_terms", "Subject terms / keywords field")]
+    + [_v2vl(_EF, "source_field", "abstract", "Abstract field")]
+    + [_v2vl(_EF, "source_field", "full_text", "Full text field")]
+    # semantic_signals
+    + [_v2vl(_SS, "axis_group", c, l) for c, l in _AXIS_GROUP_LABELS]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)